app/tests/test_views.py

```diff
@@ -0,0 +1,77 @@
+from flask.ext.testing import TestCase
+
+from app.factory import create_app, user_datastore
+from app.models import User
+
+from .test_models import DbTestCase
+
+LOGGED_IN_SENTINEL = '<a href="/me"'
+
+class ViewTestCase(DbTestCase):
+    def create_app(self):
+        config = self.BASE_APP_CONFIG.copy()
+        config.update(
+            WTF_CSRF_ENABLED=False,
+            SECURITY_REGISTERABLE=True,
+            SECURITY_SEND_REGISTER_EMAIL=False,
+            CACHE_NO_NULL_WARNING=True,
+            SECRET_KEY='test',
+            S3_BUCKET_NAME='test_bucket'
+        )
+        return create_app(config=config)
+
+    def register_and_login(self, username, password):
+        res = self.client.post('/register', data=dict(
+            next='/',
+            email=username,
+            password=password,
+            password_confirm=password,
+            submit='Register'
+        ), follow_redirects=True)
+        self.assert200(res)
```

Comment:
Hmm, I'm not seeing the reasoning... Though I was thinking about having an `assert_success=True` arg to these methods so the assertion could be disabled if a test didn't want the assertion.